sdk/eventhub/perf-tests/event-hubs-track-1/README.md

```diff
@@ -0,0 +1,12 @@
+### Guide
+
+1. Navigate to `sdk\eventhub\perf-tests\event-hubs-track-1`
+2. Do `rush update`.
+3. Run `npm run setup`.
+4. Create an event-hubs namespace and populate the .env file with `EVENTHUB_CONNECTION_STRING`, `EVENTHUB_NAME` and `CONSUMER_GROUP_NAME` variables.
+5.  Run the tests as follows.
+   - batch send
+     - `npm run perf-test:node -- SendTest --warmup 2 --duration 7 --parallel 2`
+   - receive(Standalone test - doesn't use the framework)
+     - `ts-node test/receive.spec.ts [eventBodySize] [numberOfEvents]`
+     - `ts-node test/receive.spec.ts 1024 10000`
```

Comment:
nit: I’m confused by the letter casing and use of parenthesis in this list (not in the code segments). receive( looks like a function, batch send looks like a bash line.

Reply:
The receive method doesn't use the perf framework.
Mike is working on a framework for the event based SDKs, for the receiving part, we'll migrate to that once we have the framework for the receive tests.
Until then, we'll use these standalone tests for comparison.

It'll be expensive to convert the receive test to have a similar command, without much returns as we are anyway moving to the new framework from Mike.

Reply:
No no, I don’t mean that, I mean in how the text is written in this file! I would expect something more like:

```
5. Run the tests as follows:
  - To test sending requests in batch: `npm run …`.
  - To test the `receive` method (this test does not use the framework): `ts-node ….`, where an example would be `ts-node …`.
  ```

Reply:
Ah, I was lazy. 🥱

Reply:
Updated